### PR TITLE
[JavaScript] : Play Media/Video on Tap of the Poster/Thumbnail in Mobile as well as …

### LINKED ISSUE #3140: [JavaScript] : Video should play on Tap of a Thumbnail on Desktop as well as Mobile
# Platform

What platform is your issue or question related to? (Delete other platforms).
* JavaScript

# Author or host
Host

# Version of SDK
AdaptiveCards Release/1.1

# Issue
In the adaptive cards release version 1.1, the videos are played onClick of PlayButton. User should be allowed to play the video on Tap of Thumbail/Poster on Large Screen as well as in Mobile Devices

### PR DESCRIPTION
**Pull request by cmevawala**
…in Large Desktop

Fixes #3140 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/3141)

### CODE REVIEW

**Comment by dclaux** [comment]
Thanks for your contribution. Could you detail your change a little more? Is it just about allowing playback if clicking anywhere on the poster rather than just the Play button? Is this introducing any visual change?

You do seem to have a couple indentation issues.

**Comment by matthidinger** [comment]
This PR implements #3140

**Comment by dclaux** [comment]
Is this the behavior in other renderers? Do we need to make a similar change everywhere?

**Comment by cmevawala** [comment]
@dclaux : It is all about playing a video by tapping on a poster. It has not introduced any visual changes. I will update the indentation issues. We are using this library only for Web and not for any other platforms(iOS/Android) hence could't got a chance to review them.

**Comment by cmevawala** [comment]
@dclaux : I have updated this PR. Could you please approve and merge this PR?

**Comment by cmevawala** [comment]
@matthidinger, @dclaux : Could you please merge this PR?

**Comment by matthidinger** [comment]
This change seems good to me, but @dclaux is on vacation for a few weeks. I will test it and run it by the rest of the team but we may not be able to merge/release until he's back. Will keep this PR posted with any updates

**Comment by cmevawala** [comment]
@matthidinger : Any updates on this PR?

**Comment by matthidinger** [comment]
I apologize for the delay. I'd really like to review the change with @dclaux before proceeding, he is expected back from vacation next week, so not too much longer hopefully

**Comment by cmevawala** [comment]
@dclaux : appreciate if you can point the place where indentation is missing?

**Comment by dclaux** [comment]
I'm just looking at the changes in GitHub. Seeing this:
image

**Comment by cmevawala** [comment]
@dclaux : This is really strange. I don't see any such indentation issue on my side. 
image

**Comment by cmevawala** [comment]
@dclaux : This feature is a commitment for this quarter release? Need your help.

**Comment by cmevawala** [comment]
@dclaux : Any Updates?

**Comment by shalinijoshi19** [comment]
Hi @cmevawala, David is OOF this week and should be able to help once he's back on Monday next week. Meanwhile, have you been able to look at the CI failures associated with this PR to rootcause the failures? You'd be blocked until the buddy builds are green, just FYI

File: source/nodejs/adaptivecards/src/card-elements.ts [hunk 1]
```diff
@@ -1834,7 +1819,22 @@ export class Media extends CardElement {
...
+		    this.renderedElement.innerHTML = "";
+		    this.renderedElement.appendChild(mediaPlayerElement);
+
+		    mediaPlayerElement.play();
+		}
+		else {
+		    if (Media.onPlay) {
+			Media.onPlay(this);
+		    }
+		}
```
Comment (shalinijoshi19) [inline source/nodejs/adaptivecards/src/card-elements.ts]: Your indentation is off here -- can you take a look ? Make sure you have enabled viewing whitespace changes in  your editor/settings.

**Comment by shalinijoshi19** [comment]
Ping @cmevawala

**Comment by paulcam206** [comment]
closing out old PRs - feel free to revive and update this PR and resubmit for review

**Pull request closed without merging**